games.json

```diff
@@ -3199,7 +3201,8 @@
         "status": "Broken",
         "reference": "",
         "anticheats": [
-            "miHoYo Protect"
+            "miHoYo Protect 2 (Phasing Out)",
+            "Anti-Cheat Expert"
```

Comment:
Any references for this AntiCheat?

Reply:
Only this: https://intl.cloud.tencent.com.cn/products/ace

Reply:
Does it make sense to list the AntiCheat here since it's a mobile online AntiCheat?

I think we should only list those relevant to PC

Reply:
Honkai PC does have ACE: https://steamdb.info/depot/1671201/

Reply:
> Honkai PC does have ACE: https://steamdb.info/depot/1671201/

Ah good to know, thanks, was judging by the description on the link earlier which looked like it was a mobile AC